src/rules/adjacentOverloadSignaturesRule.ts

```diff
@@ -20,14 +20,25 @@ import * as ts from "typescript";
 
 import * as Lint from "../index";
 
+const IGNORE_ACCESSORS = "ignore-accessors";
+
+interface Options {
+    ignoreAccessors: boolean;
+}
+
 export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "adjacent-overload-signatures",
         description: "Enforces function overloads to be consecutive.",
-        optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+        optionsDescription: Lint.Utils.dedent`
+            If \`${IGNORE_ACCESSORS}\` is specified, then getters and setters are not considered to be overloads
+            of function with same signature.`,
+        options: {
+            type: "string",
```

Comment:
this suggests that the option is required (but only accepts one value). can you please confirm the usage here?

Reply:
@giladgray in fact, this probably should be an array, like in [memberAccessRule.ts#L63](https://github.com/palantir/tslint/blob/master/src/rules/memberAccessRule.ts#L63)